dubbo-remoting/dubbo-remoting-http12/src/main/java/org/apache/dubbo/remoting/http12/h2/Http2ServerChannelObserver.java

```diff
@@ -37,6 +37,8 @@ public class Http2ServerChannelObserver extends AbstractServerHttpChannelObserve
 
     private boolean autoRequestN = true;
 
+    private boolean closed = false;
+
```

Comment:
**Suggestion**: Ensure thread safety for the "closed" flag access and updates, considering possible concurrent modifications. <br> **Code Suggestion**: 
 ```
 +import java.util.concurrent.atomic.AtomicBoolean;
 -private boolean closed = false;
 +private AtomicBoolean closed = new AtomicBoolean(false);
 ```